### PR TITLE
 Fix/4274 write nicer html emails #4274

### PR DESCRIPTION
**Pull request by RAISEMEUP007**

### CODE REVIEW

File: packages/ui/src/app/pages/Proposals/EmailNotification.tsx [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+        <ModalOverlay>
+          <ModalContainer>
+            <ModalHeader>
+              <Link>
+                <img src={NotificationLogo} alt="Mark" />
+              </Link>
+            </ModalHeader>
+            <ModalContent>
+              <ContentTitle>Hi {'{name}'},</ContentTitle>
+              <ContentBody>New proposal {'{proposal name}'} is created... there are 2 days left to vote...</ContentBody>
```
Comment (chrlschwb) [inline packages/ui/src/app/pages/Proposals/EmailNotification.tsx:34]: According to the issue https://github.com/Joystream/pioneer/issues/4274
you must create six variations of notifications. Please create the text of the contentbody and button as variables.

**Review by chrlschwb — CHANGES_REQUESTED** [review]

**Review by chrlschwb — COMMENTED** [review]
You must define "EmailNotifications.tsx" inside "src/common"

**Review by thesan — CHANGES_REQUESTED** [review]
@AnttoLion I don't know if this PR is still in progress, but in case it is:

Using React to generate html emails looks like a bad idea. Remember that the emails are generated on the backend so there shouldn't be any changes in `packages/ui/*`. Also you can't run js in emails and even though React can be used for ssr it's really not the right tool when the frontend can't run React.

**Comment by kdembler** [comment]
@thesan This is superseded by my PR, can be closed

**Comment by thesan** [comment]
Done in #4511

**Pull request closed without merging**